.travis.yml

```diff
@@ -40,12 +40,13 @@ script:
 
 env:
   global:
-    - NODE_PATH=/usr/local/lib/node_modules
-    - NODE_TLS_REJECT_UNAUTHORIZED=0
-    - FAKE_CERT_PATH=~/build/TwilioDevEd/api-snippets/twilio-api-faker/keystore/twilio_fake.pem
-    - GRADLE_OPTS="-Dorg.gradle.daemon=true"
-    - BASH=/bin/bash
-    - BASH_SOURCE=/usr/local/bin/virtualenvwrapper.sh
+  - NODE_PATH=/usr/local/lib/node_modules
+  - NODE_TLS_REJECT_UNAUTHORIZED=0
+  - FAKE_CERT_PATH=~/build/TwilioDevEd/api-snippets/twilio-api-faker/keystore/twilio_fake.pem
+  - GRADLE_OPTS="-Dorg.gradle.daemon=true"
+  - BASH=/bin/bash
+  - BASH_SOURCE=/usr/local/bin/virtualenvwrapper.sh
+  - PARALLEL=4
```

Comment:
Lets maintain the proper indentation on the `*.yml`, to make it more readable and consistent.